model.py

```diff
@@ -7356,6 +7357,11 @@ class DeliveryMechanism(Base):
     STREAMING_DRM = "Streaming"
     OVERDRIVE_DRM = "Overdrive DRM"
 
+    STREAMING_PROFILE = ";profile=http://librarysimplified.org/streaming"
```

Comment:
I realize I told you to call it this but it was in the way of a generic example. I think `http://librarysimplified.org/terms/profiles/streaming-media` would be better for the long term. It fits in with our link relations, genre names, etc.